main.go

```diff
@@ -505,6 +558,30 @@ func getDefaultPort() (port string, err error) {
 		portPath = "/dev/cu.usb*"
 	case "linux":
 		portPath = "/dev/ttyACM*"
+	case "windows":
+		cmd := exec.Command("wmic", "PATH", "Win32_SerialPort",
+			"get", "DeviceID")
```

Comment:
Did you see this?
http://a46554.blogspot.com/2018/05/go-lang-list-all-com-port-in-windows.html?m=1
Found it with a quick Google search. Not sure whether it works, but it might avoid an external command.

Reply:
We are already using the `wmic` command which is part of Windows itself. The command seems like the proper way to do this vs. search for some Windows registry entries that might change or even be entirely different based on system config.